app/serializers/limited_notice_serializer.rb

```diff
@@ -0,0 +1,47 @@
+class NoticeSerializer < ActiveModel::Serializer
```

Comment:
Yep, I caught that too as I was struggling with getting the test I'm building for this to work.